src/main/resources/application.yml

```diff
@@ -21,3 +21,6 @@ management:
         exposure.include: "*"
       web:
         exposure.include: info, health
+
+app:
+  brokerEncryptPassword: \${BROKER_ENCRYPT_PASSWORD}
```

Comment:
let's configure on the helm project the configuration of this variable

Reply:
done